airflow/example_dags/tutorial_taskflow_api_etl.py

```diff
@@ -45,14 +46,8 @@
 # [END default_args]
 
 # [START instantiate_dag]
-with DAG(
-    'tutorial_taskflow_api_etl_dag',
-    default_args=default_args,
-    description='TaskFlow API ETL DAG tutorial',
-    schedule_interval=None,
-    start_date=days_ago(2),
-    tags=['example'],
-) as dag:
+@dag(default_args=default_args, schedule_interval=None, start_date=days_ago(2))
+def tutorial_taskflow_api_etl():
```

Comment:
Documentation in `@dag` decorator should  be a function `__doc__`
```suggestion
def tutorial_taskflow_api_etl():
  """### TaskFlow API Tutorial Documentation
  This is a simple ETL data pipeline example which demonstrates the use of the TaskFlow API
  using three simple tasks for Extract, Transform, and Load.
  Documentation that goes along with the Airflow TaskFlow API tutorial located
  [here](https://airflow.apache.org/docs/stable/tutorial_taskflow_api.html)
  """
```

Reply:
+1